src/components/MovieDetails.js

```diff
@@ -10,10 +10,25 @@ class MovieDetails extends Component {
     this.state = {
       id: props.currentMovie,
       currentMovie: '',
+      videos: [],
+      currentVideo: ''
```

Comment:
Since currentVideo will ultimately hold an object, let's change this from an empty string